docs/source/shared/download-schema.mdx

```diff
@@ -3,7 +3,7 @@ Apollo Android requires your GraphQL server's schema as either an introspection
 
 > **Note**: If you don't have a GraphQL server yet, you can use the server from the [tutorial](https://www.apollographql.com/docs/android/tutorial/00-introduction/): https://apollo-fullstack-tutorial.herokuapp.com/graphql.
 
-The Apollo Gradle plugin exposes a `downloadApolloSchema` task to help you obtain your schema. Provide this task your server's GraphQL endpoint and the output location for the `schema.json` file:
+The Apollo Gradle plugin exposes a `downloadApolloSchema` task to help you obtain your schema. Provide this task your server's GraphQL endpoint and the output location for the `schema.graphqls` file:
```

Comment:
I just pushed a small patch to change occurences of `schema.json` to `schema.graphqls` here and in a couple of other places too.

Reply:
Thank you!